Bottom: a5732bc273c3630a7809fcc37ea3b1167f0a17c7
Top:    f6ed14f0d4c074372e06da0fc0ae8e7d128f67f1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-27 00:30:41 +0100

Refresh of comment-out-check-for-gui-hid-

---

diff --git a/src/draw.c b/src/draw.c
index cec5372..b263b9f 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -188,7 +188,7 @@ DrawStrippedText (ElementTypePtr Element, int min_width)
 static void
 draw_element_name (ElementType *element)
 {
-  if ((TEST_FLAG (HIDENAMESFLAG, PCB) && gui->gui) ||
+  if ((TEST_FLAG (HIDENAMESFLAG, PCB) /*&& gui->gui*/) ||
       TEST_FLAG (HIDENAMEFLAG, element))
     return;
   if (doing_pinout || doing_assy)
diff --git a/src/hid/ps/ps.c b/src/hid/ps/ps.c
index bdcf79b..06bffa0 100644
--- a/src/hid/ps/ps.c
+++ b/src/hid/ps/ps.c
@@ -1056,9 +1056,9 @@ ps_draw_arc (hidGC gc, int cx, int cy, int width, int height,
 	  cx, cy, width, height, start_angle, delta_angle, sa, ea);
 #endif
   use_gc (gc);
-  pcb_fprintf (global.f, "%d %d %mi %mi %mi %mi %mi a\n",
+  pcb_fprintf (global.f, "%d %d %mi %mi %mi %mi %g a\n",
                sa, ea, -width, height, cx, cy,
-               (global.linewidth + 2 * global.bloat) / width);
+               (double) (global.linewidth + 2 * global.bloat) / (double) width);
 }
 
 static void
